src/einsteinpy/utils/exceptions.py

```diff
@@ -1,5 +1,29 @@
+"""Docstring for exceptions.py module
+
+This module defines the BaseError class which is the base class for all \
+custom Errors in EinsteinPy and the CoordinateError class which is a child class\
+used for raising exceptions when the geometry does not support \
+the coordinate system.
+
+"""
+
+
 class BaseError(Exception):
+    """
+    Base Error class for custom errors.
+
+    """
+
     def __init__(self, *args, **kwargs):
+        """
+        Constructor
+        Joins the list of args into the message string
```

Comment:
`Joins ``args`` into a ``message`` string` and have a blank line after.

Reply:
Also, `sphinx` would interpret L19 and L20 as a single line. Use line-blocks to force newline, like so:
```
| Constructor
| Joins ``args`` into a ``message`` string
```